src/data/dynamodb.py

```diff
@@ -0,0 +1,291 @@
+import contextlib
+import datetime
+from collections.abc import Iterable
+from uuid import uuid4
+
+import boto3
+
+from data.persistence import DBResult, SortMethod, TilePoolDB, tile_to_dict
+from game.game import Tile, TilePool
+
+
+class DynamoTilePoolDB(TilePoolDB):
+    def __init__(self, table_name: str = "BingoMaker"):
+        self.client = boto3.client("dynamodb", region_name="us-east-1")
+        self.table_name = table_name
+
+        with contextlib.suppress(Exception):
+            self.client.create_table(
+                TableName=table_name,
+                KeySchema=[{"AttributeName": "id", "KeyType": "HASH"}],
+                AttributeDefinitions=[{"AttributeName": "id", "AttributeType": "S"}],
+                ProvisionedThroughput={"ReadCapacityUnits": 1, "WriteCapacityUnits": 1},
+            )
+
+    def _iter_data(self) -> Iterable[DBResult]:
+        print("Called _iter_data()")
```

Comment:
I'm confused what the purpose of this method is. It seems be called when sorting results, but doesn't actually yield an iterable of DBResult

Reply:
this was left in when i copied the skeleton from another one of the drivers

Reply:
removed in 0a6eb6e